json-endpoints/class.wpcom-json-api-list-media-v1-1-endpoint.php

```diff
@@ -97,6 +97,20 @@ function callback( $path = '', $blog_id = 0 ) {
 		}
 
 		$response = array();
+
+		// @start-hide-in-jetpack                   
```

Comment:
Probably should be wrapped in `IS_WPCOM` constant check -- these comments to block out content from Jetpack only work for JS files, sadly.